src/util/os_utils.js

```diff
@@ -341,12 +340,62 @@ function get_disk_mount_points() {
         });
 }
 
+function _df_parse(stdout) {
+    return stdout
+            .trim()
+            .split(/\r\n|\r|\n/) // split into rows
+            .slice(1) // strip column headers away
+            .map(function(row) {
+                const columns = row
+                // one or more whitespaces followed by one or more digits
+                // must be interpreted as column delimiter
+                .replace(/\s+(\d+)/g, '\t$1')
+                // one or more whitespaces followed by a slash
+                // must be interpreted as the last column delimiter
+                .replace(/\s+\//g, '\t/')
+                // split into columns
+                .split(/\t/);
+
+        return {
+            filesystem: columns[0],
+            size: parseInt(columns[1], 10),
+            used: parseInt(columns[2], 10),
+            available: parseInt(columns[3], 10),
+            capacity: parseInt(columns[4], 10) * 0.01,
+            mount: columns.slice(5).join(' '),
+        };
+    });
+}
+
+async function df(file_path, callback, flags = '') {
```

Comment:
It is only meant to be used here (yes, and tests) 
we should call it with a private name convention 
```suggestion
async function _df(file_path, callback, flags = '') {
```